.gitignore

```diff
@@ -13,6 +13,7 @@ config/database.yml
 # Ignore sqlite3 databases
 **/*.sqlite*
 
-# Ignore and env files
+# Ignore Guard and dotenv files
+Guardfile
```

Comment:
Is there any advantage to not having a Guardfile in the repo? File doesn't change much and it saves everyone the hassle

I love guard :)

Also, zeus works way better than spring :) and is installed outside the repo :)